texdoc/__init__.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+
+"""texdoc extension
+
+This is an extension to search for LaTeX documentation.
+
+Synopsis: <trigger> <query>"""
+
+import re
+import subprocess
+
+from shutil   import which
+from pathlib  import Path
+from albertv0 import *
+
+__iid__          = 'PythonInterface/v0.3'
+__prettyname__   = 'TeXdoc'
+__version__      = '1.0'
+__trigger__      = 'td'
+__author__       = 'Florian Adamsky'
```

Comment:
please use your github account for easy mentions. regards

Reply:
```suggestion
__author__       = 'Florian Adamsky (@cit)'
```